DB SCHEMA

```diff
@@ -0,0 +1,49 @@
+--create the database
+DROP DATABASE IF EXISTS hu_social;
+CREATE DATABASE hu_social;
+
+--select the database
+USE hu_social;
+
+--create tables
+CREATE TABLE users
+(
+  hu_id INT NOT NULL PRIMARY KEY,
+  hu_email VARCHAR(500) NOT NULL UNIQUE,
+  first_name VARCHAR(200) NOT NULL,
+  last_name VARCHAR(200) NOT NULL
+);
+
+CREATE TABLE login
+(
+  username VARCHAR(500) NOT NULL,
+  password VARCHAR(20) NOT NULL,
```

Comment:
See [bcrypt](https://www.php.net/manual/en/function.password-hash.php), this should be much larger than 20 characters.